core/src/main/java/io/opencensus/tags/TagContext.java

```diff
@@ -47,30 +49,39 @@
   }
 
   /**
-   * Returns a builder based on this {@code TagContext}.
+   * Returns the current {@code TagContext}.
    *
-   * @return a builder based on this {@code TagContext}.
+   * @return the current {@code TagContext}.
    */
-  public Builder toBuilder() {
-    return new Builder(getTags());
+  public static TagContext getCurrentTags() {
+    return CurrentTagsUtils.getCurrentTagContext();
   }
 
   /**
-   * Returns an empty {@code TagContext}.
+   * Returns a new empty {@code Builder}.
    *
-   * @return an empty {@code TagContext}.
+   * @return a new empty {@code Builder}.
    */
-  public static TagContext empty() {
-    return EMPTY;
+  public static Builder emptyBuilder() {
```

Comment:
I changed the name to `emptyBuilder` to distinguish it from the method that creates a builder from the current tags.  @dinooliva thought that adding tags to the current context would be the most common use case and that we should make it clear when methods ignore the current context.